source/manual/uptime-metrics.html.md

```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#content-api"
+title: Uptime Metrics
+section: Monitoring
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-08-25
+review_in: 6 months
+related_applications: [publishing-api, content-store]
+---
+
+Uptime metrics are collected for the `content-store` and the `publishing-api`,
+they are available as [a Grafana dashboard][grafana-dashboard].
+
+![Uptime Metrics](images/uptime-metrics.png)
+
+They are available broken down into a day by day view, highlighted in different
+colours representing the level of uptime. Green means 100%, orange means above
+90% and red for everything else.
```

Comment:
I think we'd probably want to adjust these guages - I'd imagine orange is less than 5 minutes, whereas 90% equates to 144 minutes. So for it to be 5 minutes I think it'd be 99.7% and above (if my Maths is right - which it probably isn't)

Reply:
Yes, I agree that 90% is too low. I was wondering if 99.7% was too high though, I would tempted to go to 10 minutes.

Reply:
Yeah that's cool - we'll get a better feel for it anyway as the data builds up. At the mo I'm not really imagining we'll see it slip below 100% much. Is that 99.3%?

Wonder if it's hard to represent it as minutes. We could do `86400 - (value * 86400)` to get seconds but we'd need grafana to have a method to show the seconds as minutes.

Reply:
Yes, that would be 99.3%.

Looks like maths operators are not yet supported in Graphite directly: https://github.com/graphite-project/graphite-web/issues/295 I'll see if we can do it in Grafana, but I suspect not.